### PR TITLE
Updated service config flags according to deploy doc

### PR DESCRIPTION
**Pull request by GPaoloni**
Post deploy update feature flags according to https://techmatters.app.box.com/notes/1370948524020.l

### CODE REVIEW

File: twilio-iac/helplines/cl/configs/service-configuration/staging.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
     "attributes": {
         "feature_flags": {
-            "enable_fullstory_monitoring": true
+            "enable_fullstory_monitoring": true,
+            "enable_client_profiles": true,
+            "enable_case_merging": true
```
Comment (robert-bo-davis) [inline twilio-iac/helplines/cl/configs/service-configuration/staging.json:6]: We should make sure to add new feature flags (set to false) to the top level common service configuration so we have a master list. I think `enable_client_profiles` also missed the top level.

Comment (GPaoloni) [reply twilio-iac/helplines/cl/configs/service-configuration/staging.json:6]: @robert-bo-davis Just added them to what I think is the right place. Can you confirm please?

Comment (robert-bo-davis) [reply twilio-iac/helplines/cl/configs/service-configuration/staging.json:6]: Yup! looks great, thanks

**Review by robert-bo-davis — APPROVED** [review]
lgtm

**Pull request merged**